pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -177,7 +177,8 @@ public SimpleLoadManagerImpl(PulsarService pulsar) {
         this.realtimeResourceQuotas.set(new HashMap<>());
         this.realtimeAvgResourceQuota = new ResourceQuota();
         placementStrategy = new WRRPlacementStrategy();
-        lastLoadReport = new LoadReport();
+        lastLoadReport = new LoadReport(pulsar.getWebServiceAddress(), pulsar.getWebServiceAddressTls(),
```

Comment:
That PR is merged, so you can just rebase to get rid of the commit from this PR.